.travis.yml

```diff
@@ -36,6 +37,10 @@ cache:
 
 jobs:
   include:
+    - stage: format
+      env: TEST_TYPE=format
+      script:
+        - git diff --name-only $TRAVIS_COMMIT_RANGE | scripts/pyfile_exists.py | xargs black -t py36 -l 100 --check
```

Comment:
woaaa (via https://docs.travis-ci.com/user/environment-variables/), that's nice!
```
TRAVIS_COMMIT_RANGE: The range of commits that were included in the push or pull request. (Note that this is empty for builds triggered by the initial commit of a new branch.)
```

Reply:
It's not explicit about this in the description, but for PR builds, it'll give you all the files that differ between `master` and the pull request. For branch builds, it'll just give you the files that have changed since the last build.